packages/react-heartwood-components/src/components/Modal/components/ModalBody/ModalBody.js

```diff
@@ -10,13 +10,22 @@ type Props = {
 	/** Optional class name */
 	className?: string,
 
+	/** Should the content go to the edge of the modal body? */
+	fullBleed?: boolean,
```

Comment:
Open to y'all's thoughts on this, but do we want to standardize around a format for this type of prop? Like `isWhateverMod`, so in this case `isFullBleed`? I know we're doing it a lot with `isSmall` and `isFullWidth` in other components.

Reply:
@jtini and @gcpantazis  yeah I think we should keep it consistent with the `isWhatever` format for boolean props. Gets kind of verbose

Reply:
Just carrying over convo from slack: 

> Yeah was just thinking about that. Not sure. I like the be-participle prefixed attributes when its a state. Not sure for something like this

> Like `isSmall` not sure I like but `isLoading` I definitely like

> But I think that's because the latter is something stateful... its maybe a good indicator of what is and isn't stateful. :thinking_face: